app.py

```diff
@@ -0,0 +1,26 @@
+import boto3
+from botocore.config import Config
+
+from conf import settings
+from entities.connection import DynamoDBDriver
+
+
+def test():
+    conf = Config(retries={"max_attempts": 1, "mode": "standard"})
+
+    conn = boto3.resource(
```

Comment:
And we should store connection in module scope in order not to create connection every time when we call a lambda function